pkg/cmd/server/origin/master.go

```diff
@@ -164,6 +164,9 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 	handler := c.versionSkewFilter(safe)
 	handler = c.authorizationFilter(handler)
 	handler = c.impersonationFilter(handler)
+	if c.BasicAudit {
+		handler = basicAuditHandler(handler, c.getRequestContextMapper())
```

Comment:
I prefer the check like `versionSkewFilter` where if the config says "no check" it just returns the same handler back.  Keeps this method easier to read.